examples/pytorch/hetgnn/evaluation.py

```diff
@@ -12,16 +12,16 @@ def load_data(data_file_name, n_features, n_samples):
         data_file = csv.reader(f)
         data = numpy.empty((n_samples, n_features))
```

Comment:
Import numpy as np then replace numpy. with np., this way this code can be better copied into other people's code.

Reply:
That's the original code..